src/ActionParser.h

```diff
@@ -30,22 +32,23 @@ namespace snowcrash {
     /** Action Definition Type */
     enum ActionType {
         NotActionType = 0,
-        DependentActionType,      /// Action isn't fully defined, depends on parents resource URI
-        CompleteActionType,       /// Action is fully defined including its URI
+        DependentActionType, /// Action isn't fully defined, depends on parents resource URI
+        CompleteActionType, /// Action is fully defined including its URI
         UndefinedActionType = -1
     };
 
     /**
      * Action Section processor
      */
-    template<>
+    template <>
     struct SectionProcessor<Action> : public SectionProcessorBase<Action> {
 
         static MarkdownNodeIterator processSignature(const MarkdownNodeIterator& node,
-                                                     const MarkdownNodes& siblings,
-                                                     SectionParserData& pd,
-                                                     SectionLayout& layout,
-                                                     const ParseResultRef<Action>& out) {
+            const MarkdownNodes& siblings,
```

Comment:
In definition it is not so problematic, but on call:
```
Something(arg1,
                 arg2,
                 arg3);

AndNowSometingCompletlyDifferent(arg1,
                                                          arg2,
                                                          arg3);
```
In my opinion it is much more readable
```
Something(arg1,
    arg2,
    arg3);

AndNowSometingCompletlyDifferent(arg1,
    arg2,
    arg3);
```